docs: migrator for KeptnEvaluationProvider -> KeptnMetricsProvider (#945)
keptn/lifecycle-toolkit: docs/content/en/docs/tasks/migrate-keptnevaluationprovider/_index.md

---
title: Migrate KeptnEvaluationProvider to KeptnMetricsProvider
description: Migrate deprecated KeptnEvaluationProvider to KeptnMetricsProvider.
icon: concepts
layout: quickstart
weight: 20
hidechildren: true # this flag hides all sub-pages in the sidebar-multicard.html
---

## Migrate KeptnEvaluationProvider to KeptmMetricsProvider

> Due to the recent changes by splitting the `klt-operator` into two separate operators: `klt-operator` and

Reviewer: ```suggestion
Due to the recent changes by splitting the `klt-operator` into two separate operators: `klt-controller` and
```

> `metric-operator` (introduced in version 0.7.0), some of the functionality and behavior have been moved,

Reviewer: ```suggestion
`klt-metrics` (introduced in version 0.7.0), some of the functionality and behavior have been moved,
```

> changed, or renamed. The `KeptnEvaluationProvider` CRD was deprecated and replaced by the `KeptnMetricsOperator`

Reviewer: ```suggestion
changed, or renamed. The `KeptnEvaluationProvider` CRD was deprecated and replaced by the `KeptnMetricsProvider`
```

---
title: Migrate KeptnEvaluationProvider to KeptnMetricsProvider
description: Migrate deprecated KeptnEvaluationProvider to KeptnMetricsProvider.
icon: concepts
layout: quickstart
weight: 20
hidechildren: true # this flag hides all sub-pages in the sidebar-multicard.html
---

## Migrate KeptnEvaluationProvider to KeptmMetricsProvider

Due to the recent changes by splitting the `klt-operator` into two separate operators: `lifecycle-operator` and

> `metric-operator` (introduced in version 0.7.0), some of the functionality and behavior have been moved,

Reviewer 2: ```suggestion
`metrics-operator` (introduced in version 0.7.0), some of the functionality and behavior have been moved,
```
